app/src/main/java/ch/epfl/sdp/blindly/user/UserHelper.kt

```diff
@@ -106,19 +110,28 @@ class UserHelper {
         return FirebaseAuth.getInstance().currentUser?.uid
     }
 
-    //TODO refactor this and move to UserRepository
```

Comment:
Was this TODO no longer relevant, in terms of moving it to UserRepository?

Reply:
I thought a bit more about it and  I don't believe it would be very practical to move it to the UserRepository, since then it would need a @Injection userHelper, just to get the uid....
But then again it would make more sense to put it in the repo...